prow/plugins/verify-owners/verify-owners_test.go

```diff
@@ -46,22 +48,22 @@ labels:
     labels:
     - label1
 `),
-	"invalidSyntax": []byte(`approvers
+	"invalidSyntax": []byte(`approvers:
```

Comment:
Why did these have to change?

Reply:
It was to make the invalid line more to the end of the file in order to have a patch not starting with line 1.